web/src/lib/api-client/apiClient.ts

```diff
@@ -74,6 +75,10 @@ export const getCompletedFiling = (): Promise<UserData> => {
   return get(`/completed-filing`);
 };
 
+export const postTaxClearanceCertificate = (userData: UserData): Promise<TaxClearanceCertificateResponse> => {
+  return post(`/postTaxClearanceCertificate`, userData, false);
```

Comment:
Why do we need to include `false` here?

Reply:
I'll set up some time to chat about this. I want to understand why the `authConfig` function is used, and when it is appropriate to use it. I noticed it's set to false for other api calls.

Reply:
Oversight, I'll make the updates. I need to dig into how Auth works in the backend.